### PR TITLE
There are some return parameters that don't make sense in Python so don't return them

### LINKED ISSUE #635: Functions that return lists also return the length of the list without reason
```python
    voltage_measurements, current_measurements, in_compliance, actual_count = session.fetch_multiple(session.fetch_backlog)
```
from [nimi-python/nidcpower_measure_record.py at master · ni/nimi-python · GitHub ](https://github.com/ni/nimi-python/blob/master/src/nidcpower/examples/nidcpower_advanced_sequence.py#L39)

 If actual_count is the length of the other returned lists, why is it being returned?  This feels like a vestige of a C API for saying how much of a buffer was actually filled.

Note: other APIs seem to be subject to this, see https://github.com/ni/nimi-python/blob/master/src/nidmm/examples/nidmm_fetch_waveform.py#L25

### PR DESCRIPTION
**Pull request by texasaggie97**
[X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
[X] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.
[X] I've ~~added~~updated tests applicable for this pull request
### What does this Pull Request accomplish?
* Add option to code generator to allow skipping some parameters in the Python API.
  * Right now, only used for return list

### List issues fixed by this Pull Request below, if any.

* Fix #635 

### What testing has been done?
* Unit
* System

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ All notable changes to this project will be documented in this file.
   * #### Added
     * `fetch_waveform_into` for high-performance fetch using numpy.array of float64.
   * #### Changed
+    * Removed `actual_count` from `measure_multiple()` return list
```
Comment (marcoskirsch) [inline CHANGELOG.md]: Wording is weird... Please clarify

Comment (marcoskirsch) [reply CHANGELOG.md]: Aren't there other functions? fetch_waveform()?

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,6 +33,7 @@ All notable changes to this project will be documented in this file.
 * ### NI-DCPower
   * #### Added
   * #### Changed
+    * Removed `actual_number_of_points` from `fetch_waveform()` return list
```
Comment (marcoskirsch) [inline CHANGELOG.md]: Wording is weird... Please clarify

File: src/nidcpower/examples/nidcpower_measure_record.py [hunk 3]
```diff
@@ -27,9 +27,9 @@
     samples_acquired = 0
     with session.initiate():
         while samples_acquired < args.length:
-            voltage_measurements, current_measurements, in_compliance, actual_count = session.fetch_multiple(count=session.fetch_backlog)
-            assert len(voltage_measurements) == len(current_measurements) == len(in_compliance) == actual_count
-            for i in range(actual_count):
+            voltage_measurements, current_measurements, in_compliance = session.fetch_multiple(count=session.fetch_backlog)
+            assert len(voltage_measurements) == len(current_measurements) == len(in_compliance)
+            for i in range(len(voltage_measurements)):
```
Comment (marcoskirsch) [inline src/nidcpower/examples/nidcpower_measure_record.py]: This is a really good place to use zip().

```
>>> a = ['a', 'b', 'c']
>>> b = ['Marcos', 'Frank', 'Mark']
>>> zip (range(len(a)),a,b)
[(0, 'a', 'Marcos'), (1, 'b', 'Frank'), (2, 'c', 'Mark')]
```

File: src/nidcpower/metadata/functions_addon.py [hunk 4]
```diff
@@ -84,6 +84,11 @@
     'GetError':                     { 'is_error_handling': True },
 }
 
+# There are some parameters that are needed in the C function call we use under the hood, but that we do not want in the Python API
+functions_remove_from_python_api = {
+    'FetchMultiple':                { 'parameters': { 7: { 'use_in_python_api': False, }, }, },
```
Comment (marcoskirsch) [inline src/nidcpower/metadata/functions_addon.py]: Does this get populated correctly at add_all time for things that we don't specify?
For example, in `nifgen.Session.write_waveform` the size should have a value of False.

File: src/nidmm/metadata/functions_addon.py [hunk 5]
```diff
@@ -87,6 +87,11 @@
     'GetError':                     { 'is_error_handling': True, },
 }
 
+# There are some parameters that are needed in the C function call we use under the hood, but that we do not want in the Python API
+functions_remove_from_python_api = {
+    'FetchWaveform':                { 'parameters': { 4: { 'use_in_python_api': False, }, }, },
```
Comment (marcoskirsch) [inline src/nidmm/metadata/functions_addon.py]: There's more: ReadWaveform,  ReadMultipoint, FetchMultipoint

File: src/nidcpower/metadata/functions_addon.py (hunk 4 above)
Comment (texasaggie97) [reply src/nidcpower/metadata/functions_addon.py]: Added auto population for 'len' and 'ivi-dance' size parameters

File: src/nidmm/metadata/functions_addon.py (hunk 5 above)
Comment (texasaggie97) [reply src/nidmm/metadata/functions_addon.py]: Added

**Pull request merged**